keras_cv/layers/object_detection_3d/voxelization_test.py

```diff
@@ -75,15 +65,17 @@ def test_voxelization_output_shape_with_z(self):
         # (15 - (-15)) / 1 = 30
         # the last dimension is replaced with MLP dimension, z dimension is
         # skipped
-        self.assertEqual(output.shape, [1, 400, 400, 30, 20])
+        self.assertEqual(output.shape, (1, 400, 400, 30, 128))
 
     def test_voxelization_numerical(self):
-        point_net = keras.layers.Lambda(lambda x: x)
         layer = DynamicVoxelization(
-            point_net=point_net,
             voxel_size=[1.0, 1.0, 10.0],
             spatial_size=[-5, 5, -5, 5, -2, 2],
         )
+        # Make the point net a no-op to allow us to verify the voxelization.
+        layer.point_net_dense = keras.layers.Lambda(lambda x: x)
```

Comment:
Isn't there a `keras.layers.Indentity` for this?

Reply:
TIL -- thanks!